ui/src/main/webapp/wizards/sources/actions.js

```diff
@@ -1,6 +1,7 @@
 import { post } from 'redux-fetch'
 
 import { getAllConfig } from '../../reducer'
+import { getDiscoveryConfigs } from './reducer'
```

Comment:
What is the standard we use to decide whether to put in a forwarding function in the root reducer vs. referencing a function in the local reducer directly?